src/controls/Controls.spec.js

```diff
@@ -1 +1,28 @@
-// Test away!
\ No newline at end of file
+// Test away!
+import React from 'react';
+import * as rtl from 'react-testing-library';
+import 'jest-dom/extend-expect';
```

Comment:
You'll want to update these libraries to their new location since these ones were deprecated:
- `@testing-library/react`
- `@testing-library/jest-dom`